### PR TITLE
Double click on owl to change API constants

### PR DESCRIPTION
**Pull request by jmularski**
Written basic implementation of PanResponder, and used earlier redux constant store to change constants.

### CODE REVIEW

**Comment by jmularski** [comment]
Owl dialog
This should be it

File: app/containers/SignIn/index.js [hunk 1]
```diff
@@ -183,7 +198,21 @@ class SignIn extends Component {
       </Animated.View>
     );
   }
-
+  changeValues = (value) => {
+    splitted_vals = value.split(":");
+    store.dispatch(setHostUrl(splitted_vals[0]));
+    store.dispatch(setHostPort(splitted_vals[1]));
```
Comment (gayashanbc) [inline app/containers/SignIn/index.js]: Where is the store object that you are trying to access here?
Where have you imported these actions?

**Comment by jmularski** [comment]
Sure, will do, sorry about trouble, didn't test thoroughly, I'll try to send the screenshots by midnight Indian time

**Comment by jmularski** [comment]
Hey, once again sorry for pushing such bad and untested code. Lesson learned. Here are the screenshots you asked for. @gayashanbc 
set_host_port
set_host_url

**Comment by gayashanbc** [comment]
@jmularski nice work, looks fine :)

**Pull request merged**